packages/design-picker/src/components/index.tsx

```diff
@@ -301,4 +383,4 @@ const DesignPicker: React.FC< DesignPickerProps > = ( {
 	);
 };
 
-export default DesignPicker;
+export { DesignPicker as default, GeneratedDesignPicker };
```

Comment:
[nit] Prefer to separate the DesignPicker and GeneratedDesignPicker into different files to keep it simple as I think `DesignPicker` becomes larger. We could also do it in the follow-up PR.

Reply:
Agreed